app/popup.js

```diff
@@ -477,6 +484,23 @@ class OptionsTab extends React.Component {
           </tbody>
         </table>
 
+        <h4 className="page-header">Import / Export</h4>
+        <div className="row">
+          <div className="col-xs-8">
+            <Button label='Export' clickHandler={_exportData} className='glyphicon-export'/>
```

Comment:
If you preserve a space between these buttons, they look a bit more naturally laid out:

```
<Button />{' '}
<Button />
```

<img width="234" alt="screen shot 2017-06-12 at 10 25 59 am" src="https://user-images.githubusercontent.com/29612/27038632-8fa99ea0-4f59-11e7-8f75-d418a4848780.png">